package syncro.pb;

option java_package = "uk.me.grambo.syncro.pb";

message PacketHeader {
	required int32 packet_type = 1;
	repeated int32 subpacket_sizes = 2;
}
